pygmt/src/histogram.py

```diff
@@ -31,8 +31,8 @@
     c="panel",
     d="nodata",
     e="find",
-    g="gap",
     h="header",
+    i="incols",
     l="label",
     p="perspective",
     t="transparency",
```

Comment:
Need to add `i="sequence_comma"` to `kwargs_to_strings`.

Reply:
Thanks, fixed.